shopfloor_mobile_base/static/wms/src/components/detail/detail_mixin.js

```diff
@@ -85,14 +85,18 @@ export var ItemDetailMixin = {
             // Defining defaults for an Object property
             // works only if you don't pass the property at all.
             // If you pass only one key, you'll lose all defaults.
-            const opts = _.defaults({}, this.$props.options, {
```

Comment:
why this change?

Reply:
There's no good reason to use defaults here. And standard javascript has nearly all utility function that underscore/lodash provides. In the end the plan was to remove underscore from the codebase if that's possible and stick to vanilla js for object literal and array manipulation.